service/siteService.test.js

```diff
@@ -1,16 +1,119 @@
+        { name: "10k Hoodies- Brooklyn Park", neighborhood: ["North Suburbs"], address: "3217 85th Ave N, Brooklyn Park, MN 55443\n", longitude: -93.3232077, latitude: 45.1083272, mostRecentlyUpdatedAt: "2021-01-30T02:24:24.000Z", currentlyOpenForDistributing: "no", openingForDistributingDonations: "never", closingForDistributingDonations: undefined, currentlyOpenForReceiving: "yes", openingForReceivingDonations: "9:00 am", closingForReceivingDonations: "3:00 pm", urgentNeed: undefined, seekingMoney: undefined, seekingMoneyURL: undefined, noIdNeeded: undefined, someInfoRequired: undefined, warmingSite: undefined, seekingVolunteers: undefined, publicTransitOptions: [{ routeName: "89", backgroundColor: "#771473", icon: "directions_bus", distance: "(3 blocks)" }, { routeName: "Blue", backgroundColor: "#0055A5", icon: "tram", distance: "(2 blocks)" }, { routeName: "Green", backgroundColor: "#00B100", icon: "tram", distance: "(4 blocks)" }], accepting: "(as of 12/24 10:20am) Hoodies, new and packaged personal hygiene products. If you are able, please attach a small and encouraging handwritten note to the hoodie that you are donating.", notAccepting: undefined, notes: "(as of 12/24 10:20am)\nBring a hoodie to Pilgrims Dry Cleaners \nMonday - Friday 7am - 6pm\nSaturday 9am - 3pm\n\nYou can also ship a hoodie to\n \nATTN:\nBrandon Glova\nPO\nBox 8112\nMinneapolis,\nMN 55408-3103 \n\nYou may also support 10khoodies by purchasing a hoodie at https://highlovelife.com/pages/10khoodies \n\nOne year ago, DJ Bonics started a mission to turn Minnesota from the land of 10,000 Lakes to the Land of 10,000 Hoodies. The year 2020 has been met with many challenges and opportunities to grow. This project serves as a reminder that we all have something to give. By donating a sweatshirt, you are providing someone in your community with warmth and comfort that lasts beyond a Minnesota winter. ", color: "#fc03df" }]
+
+        airtableClientMock.getMutualAidSites.mockResolvedValue(airtableResponse)
+
+        let result = await siteService.getMutualAidSites("/v1/mutual_aid_sites")
+
+        expect(airtableClientMock.getMutualAidSites).toHaveBeenCalled()
+        expect(cacheServiceMock.writeCache).toHaveBeenCalled()
+
+        expect(result).toStrictEqual(expectedResult)
+    })
+})
+
+describe("transformPublicTransit", () => {
```

Comment:
I genuinely love `describe`/`it` test forms.  I know functionally it's the same as just `test(...)`, but a part of me likes reading tests as statements of fact 😆

Reply:
for sure, big fan of BDD and readable tests over here 🤓 

(didn't realize this was a syntax option in jest when I first wrote these)